VideoEncoding/CreateTransform_Job_PresetOverride/create-transform-job-preset-override.py

```diff
@@ -0,0 +1,318 @@
+  parameters = myTransform)
+
+print(f"{transform_name} created (or updated if it existed already). ")
+
+job_name = 'MyEncodingSpriteThumbnailJob'+ uniqueness
+print(f"Creating EncodingSpriteThumbnail job {job_name}")
+files = (source_file)
+
+input = JobInputAsset(asset_name=in_asset_name)
+
+print("Creating the output Asset (container) to encode the content into...")
+outputs = JobOutputAsset(asset_name=out_asset_name)
+
+print(f"Creating a new custom preset override and submitting the job to the empty transform {transform_name} job queue...")
+
+# Create a new Preset Override to define a custom standard encoding preset
+standard_preset_h264 = StandardEncoderPreset(
+    codecs = [
+        H264Video(
+            # Next, add a H264Video for the video encoding
+            key_frame_interval = timedelta(seconds=2),
+            complexity = H264Complexity.SPEED,
+            layers = [
+                H264Layer(
+                    bitrate = 3600000,  # Units are in bits per second and not kbps or Mbps - 3.6 Mbps or 3,600 kbps
+                    width = "1280",
+                    height = "720",
+                    label = "HD-3600kbps"   # This label is used to modify the file name in the output formats
+                )
+            ]
+        ), 
+        AacAudio(
+            # Add an AAC Audio Layer for the audio encoding
+            channels = 2,
+            sampling_rate = 48000,
+            bitrate = 128000,
+            profile = AacAudioProfile.AAC_LC
+        )
+    ],
+    formats = [
+        Mp4Format(filename_pattern = "Video-{Basename}-{Label}-{Bitrate}{Extension}")
+    ]
+)
+
+outputs = JobOutputAsset(asset_name=out_asset_name, preset_override=standard_preset_h264)
+
+theJob = Job(input=input,outputs=[outputs])
+
+# Submit the H264 encoding custom job, passing in the preset override defined above.
+job: Job = client.jobs.create(RESOURCE_GROUP, ACCOUNT_NAME, transform_name, job_name, parameters=theJob)
+
+# Next, we will create another preset override that uses HEVC instead and submit it against the same simple transform
+# Create a new Preset Override to define a custom standard encoding preset
+standard_preset_HEVC = StandardEncoderPreset(
+    codecs = [
+        H265Video(
+            # Next, add a H265Video for the video encoding
+            key_frame_interval = timedelta(seconds=2),
+            complexity = H264Complexity.SPEED,
+            layers = [
+                H265Layer(
+                    bitrate = 1800000, # Units are in bits per second and not kbps or Mbps - 3.6 Mbps or 3,600 kbps
+                    max_bitrate = 1800000,
+                    width = "1280",
+                    height = "720",
+                    b_frames = 4,
+                    label = "HD-1800kbps" # This label is used to modify the file name in the output formats
+                )
+            ]
+        ), 
+        AacAudio(
+            # Add an AAC audio layer for the audio encoding
+            channels = 2,
```

Comment:
when passing in parameters by name, there shouldn't be any spaces around the equals sign. This should be `channels=2,` etc

Reply:
Thanks for pointing them out. I'll have the files changed.